Xamarin.Forms.Platform.GTK/Platform.cs

```diff
@@ -58,8 +58,7 @@ internal static void DisposeModelAndChildrenRenderers(Element view)
 
 			renderer = GetRenderer((VisualElement)view);
 
-			renderer?.Dispose();
-
+			((Widget)renderer).Destroy();
```

Comment:
This was checking for null before, GetRenderer can return a null renderer.

Reply:
upps